WalletWasabi.Fluent/Views/Wallets/MusicControlsView.axaml

```diff
@@ -31,14 +31,20 @@
     </Style>
   </UserControl.Styles>
   <Border Name="PART_MainBorder" Margin="3 3 3 0"
-          Classes.IsActive="{Binding IsActive, FallbackValue=false}"
           TextBlock.FontFamily="{StaticResource MonospacedFont}"
           CornerRadius="4 4 0 0">
+    <Classes.IsActive>
+      <MultiBinding Converter="{x:Static BoolConverters.And}">
+        <Binding Path="IsActive" FallbackValue="False" />
+        <Binding Path="IsWalletBalanceZero" FallbackValue="False" Converter="{x:Static BoolConverters.Not}" />
+        <Binding Path="Wallet.KeyManager.IsWatchOnly" FallbackValue="False" Converter="{x:Static BoolConverters.Not}" />
```

Comment:
Does it make sense to replace these by:

        <Binding Path="!IsWalletBalanceZero" FallbackValue="False" />
        <Binding Path="!Wallet.KeyManager.IsWatchOnly" FallbackValue="False" />